app/views/users/_member_card.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="col-lg-6 col-sm-12">
+<div class="col-lg-6 col-sm-12 column">
```

Comment:
Instead of creating the class column you could have used `d-flex flex-column`, although I think creating a class is not that bad